docs/contributing/Contributing-To-Ignite.md

```diff
@@ -22,6 +24,39 @@ node your-ignite-repo-clone/bin/ignite generate component OrderButton
 
 In addition, we have automated tests you can run with `yarn test`.
 
+### Making PRs
+
+We appreciate your contribution! To make it easier for us to review, please make sure to add a clear title and description that explains both what the change is, and why it's useful, if applicable.
+
+Maintainers may choose to rewrite this description for our changelog, or squash your commits when merging.
+
+## For maintainers
+
+### Merging PRs
+
+When merging PRs, we need maintainers to make sure that there's appropriately formatted commit message(s) for semantic-release to document changes for the release notes and pick the next-released version for Ignite. See [Releasing Ignite](./Releasing-Ignite.md) for more info on how to format commit messages.
+
+You have two choices when merging to Ignite:
+
+1. Merge with a merge commit, keeping the original commits. In this case, if the original commits are using the semantic-release conventions, those messages will be used for the changelog and release determinations.
+   1. If everything in the PR is sufficiently documented by those commit messages, you don't need to add anything to the commit message.
+   1. If there are no change entries, you should update the merge commit message when you merge to match the expected conventions.
+1. Alternatively, if you don't want to keep the original commits, you can use a squash merge. You will need to update the merge commit message to match semantic-release conventions as required by the particular change.
+
+Not every included commit in a PR needs to be formatted this way, only enough commits to ensure relevant changes have been documented.
+
+In addition, if there's a feature branch for a future version of Ignite, you'll need to merge up `master` into this feature branch. See the next section for instructions.
+
+### Managing future version branches
+
+When we're working on a new release of Ignite, we'll create a feature branch for work that's exclusive to that version (e.g., `v10`).
+
+If such a branch exists, whenever you merge a commit, you should also merge that change up from `master`->`next-version-branch`.
+
+To do so, update your local branches, check out `next-version-branch`, and run `git merge --no-ff origin/master`. You do not need to customize this merge commit message.
+
+It's a good idea to create a pull request for this merge-up if anyone else is working on Ignite along with you, so that they can be made aware of changes to `next-version-branch` ASAP. It also us to ensure tests are run before you update the branch.
```

Comment:
Iinstead of `next-version-branch`, can we maybe use a concrete example? Like `v10`?

Reply:
Low gafo but something like `release/v10.0.0` would work

Reply:
I was specifically trying to avoid implying the existence of a particular branch or naming scheme to avoid confusing anyone who goes looking for it, since these are temporary. Do y'all feel strongly about this?

Reply:
I don't mind it the way it is, likely these are more internal discussions anyway (at least the way Ignite is maintained currently)

Reply:
Yeah, I don't feel strongly about it. It could be more precise if it were an actual example, not a blocker for merging this one.